vignettes/software-permissions.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "Software permissions and regulations"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Software permissions and regulations}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This document is not a legally binding contract but present a simplified
+overview of how this package interacts with the rest of your system, the
+internet, third-party services, etc., and may inform compliance with
+institutional policies or international legislation.
+
+Although this is not a contractual document, we take user rights seriously and
+welcome any reports of non-compliance with the principles outlined here.
```

Comment:
Nothing against retaining this line but it does suggest we might act on any reported non-compliance - it may be good to be upfront that we would not?

I'm curious how compliance with the principles below relates to any software that wraps this package, e.g., another package that simply includes `read.csv()`, and `cfr_static()`. Would that be non-compliance? Would that not conflict with the MIT license?

Reply:
> Nothing against retaining this line but it does suggest we might act on any reported non-compliance - it may be good to be upfront that we would not?

Please suggest a better phrasing if possible. The idea is that if the package behaves differently than what we have described here, it's completely unexpected and potentially a serious bug.

It could be made clearer that update would ensure that the document and the actual behaviour match. If the document simply went out of date with more recent feature, the update will modify the document but if something unexpected happens, we should definitely act on it.

> I'm curious how compliance with the principles below relates to any software that wraps this package

Wrappers around this package should write their own document since behaviour will quite likely differ. We cannot be expected to be responsible for what they are doing.

Reply:
Ah I see what you mean now - I understood this as a third party using the package to build something that does not comply.

Suggest using your own explanation: "... we take user rights seriously, and welcome any reports of instances where the behaviour of the package does not conform with the principles outlined here."